tests/unit/test_column_noise.py

```diff
@@ -56,13 +42,36 @@ def dummy_dataset():
     return pd.DataFrame({"numbers": integer_series, "characters": character_series})
 
 
-def test_generate_missing_data(dummy_dataset, user_config_path):
-    config = get_configuration(user_config_path)["decennial_census"]["zipcode"][
-        "missing_data"
-    ]
+@pytest.fixture(scope="module")
+def categorical_series():
+    return pd.Series(
+        ["CA", "WA", "FL", "OR", "CO", "TX", "NY", "VA", "AZ", "''"] * 100_000, name="state"
+    )
+
+
+@pytest.fixture(scope="module")
+def default_configuration():
+    return get_configuration()
+
+
+def test_generate_missing_data(dummy_dataset):
+    # TODO: [MIC-3910] Use custom config (MIC-3866)
```

Comment:
This has been done (dummy_dataset now makes a copy of the default config and saves to tmpdir to be used as a user config)